src/_scss/pages/search/filters/recipient/_recipientType.scss

```diff
@@ -45,11 +45,13 @@
     cursor: pointer;
 
     input {
-      cursor: pointer;
+      @include usa-input-checkbox;
+      margin: 0 rem(8) 0 0;
+      appearance: auto;
```

Comment:
It looks like this line is making the checkboxes in this section look different than the others on the page - and in Edge the bg color is not appearing at all. Both of these issues were resolved when I removed this line.

Reply:
@brianpetway Could you attached a screenshot?  I'm not seeing the issue on Chrome